fabtests/common/shared.c

```diff
@@ -146,7 +146,11 @@ struct test_size_param test_size[] = {
 	{ 1 << 23, 0 },
 };
 
-const unsigned int test_cnt = (sizeof test_size / sizeof test_size[0]);
+unsigned int test_cnt = (sizeof all_test_size / sizeof all_test_size[0]);
+
+struct test_size_param * test_size = all_test_size;
+/* range of messages is dynamically allocated */
+struct test_size_param * range_test_size = NULL;
```

Comment:
nit: Above definitions should have the '*' next to the variable name - no space in between.  Yes, when you're reading code for hours on end each day, little things like this matter.

global vars will be initialized to 0/NULL, so no need to set.  Harmless though if you want to keep it.